translate/convert/test_po2web2py.py

```diff
@@ -68,6 +68,20 @@ def test_ordering_serialize(self):
 'baz': 'zab',
 'foo': 'oof',
 }
+'''
+        web2py_out = self.po2web2py(input_po)
+        assert web2py_out == expected_web2py
+
+            def test_markmin(self):
```

Comment:
Messed up indentation?

```suggestion
    def test_markmin(self):
```